sopel/irc.py

```diff
@@ -102,7 +102,7 @@ def log_raw(self, line, prefix):
                 stderr('%s %s' % (str(e.__class__), str(e)))
                 stderr('Please fix this and then run Sopel again.')
                 os._exit(1)
-        f = codecs.open(os.path.join(self.config.core.logdir, 'raw.log'),
+        f = codecs.open(os.path.join(self.config.core.logdir, self.config.basename + 'raw.log'),
```

Comment:
For consistency with the others:

```suggestion
        f = codecs.open(os.path.join(self.config.core.logdir, self.config.basename + '.raw.log'),
```